elements/lisk-p2p/src/p2p.ts

```diff
@@ -260,8 +262,36 @@ export class P2P extends EventEmitter {
 		codec.addSchema(this._rpcSchemas.peerInfo);
 		codec.addSchema(this._rpcSchemas.nodeInfo);
 
+		this._networkStats = {
+			startTime: Date.now(),
+			incoming: {
+				count: 0,
+				connects: 0,
+				disconnects: 0,
+			},
+			outgoing: {
+				count: 0,
+				connects: 0,
+				disconnects: 0,
+			},
+			banning: {
+				bannedPeers: {},
+				totalBannedPeers: 0,
+			},
+			totalErrors: 0,
+			totalPeersDiscovered: 0,
+			totalRemovedPeers: 0,
+			totalMessagesReceived: {},
+			totalRequestsReceived: {},
+		};
+
 		// This needs to be an arrow function so that it can be used as a listener.
 		this._handlePeerPoolRPC = (request: P2PRequest): void => {
+			if (!this._networkStats.totalRequestsReceived[request.procedure]) {
+				this._networkStats.totalRequestsReceived[request.procedure] = 0;
+			}
+
+			this._networkStats.totalRequestsReceived[request.procedure] += 1;
```

Comment:
A suggestion if you like this style... 

```js
this._networkStats.totalRequestsReceived[request.procedure] = ++this._networkStats.totalRequestsReceived[request.procedure] || 1;
```

Reply:
Changed it to logical op, not using unary though as it throws lint error